tests/components/trend/test_binary_sensor.py

```diff
@@ -33,6 +33,23 @@ async def _setup_component(hass: HomeAssistant, params: dict[str, Any]) -> None:
     await hass.async_block_till_done()
 
 
+@pytest.mark.usefixtures("config_entry")
```

Comment:
I dont see the need for this mark. Its using the fixture as a parameter amready.

Reply:
Oh, indeed, thanks for the review